packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison-node.vue

```diff
@@ -1,9 +1,13 @@
 <template>
 	<section>
-		<tera-operator-placeholder v-if="!node.inputs[0].value || !node.inputs[1].value" :node="node">
-			Attach models to compare
-		</tera-operator-placeholder>
-		<Button v-else label="Open" @click="emit('open-drilldown')" severity="secondary" outlined />
+		<tera-operator-placeholder :node="node"> Attach models to compare </tera-operator-placeholder>
+		<Button
+			v-if="node.inputs[0].value && node.inputs[1].value"
+			label="Open"
+			@click="emit('open-drilldown')"
+			severity="secondary"
+			outlined
+		/>
```

Comment:
I "think" you just need to have at least 2 input models, and not the first 2 input modes. eg I might have inputs[2] and inputs[3], but {0, 1} may be disconnected.

Reply:
fixed